alembic_migration/README.md

```diff
@@ -0,0 +1,41 @@
+## Table Migration
```

Comment:
Would be nice to write a small section describing the purpose of online/offline modes in Alembic and how we use them. Also, provide link to the related Alembic documentation for reference.